docs/developer-docs/integrations/sns/get-sns/local-testing.md

```diff
@@ -1,74 +1,150 @@
 
 #### 3. Ask the SNS wasm modules canister to install an SNS.
-Make a call to the SNS wasm modules canister on the local NNS 
-to request that an SNS is installed.
-To make this call, use your `dfx` identity `identityDevDfx` and
-the command as described 'here'. <!--TODO-CLI/dfx-Link: -->
-Upon receiving this call, the SNS wasm modules canister will install
-an SNS with your chosen initial parameters.
-<!--TODO-CLI/dfx-Link: once tooling is clear, make sure that here automatically
-the .yaml file is used. If this is not the case, add the information how this 
-can be ensured.-->
-
-#### 4. Add the SNS root canister as a controller to your dapp canister(s).
-To do so, use your `dfx` identity `identityDevDfx` and
-the command described 'here'.
-<!-- TODO: add this to CLI/dfx tool as need to learn SNS canisters -->
-
-#### 5. Remove all controllers other than the SNS from the dapp canister(s)
-Remove yourself, as well as any other developers,
-from the list of controllers that the dapp canister(s) have.
+Make a call to the SNS wasm modules canister on the local NNS to request that an SNS is installed.
+Installing the SNS has some preconditions:
+
+1. The SNS configuration (created in the [previous step](./preparation.md)) has to be named `sns.yml` and has to be placed in your project's root directory.
+2. Your [cycles wallet](../../../build//project-setup/cycles-wallet.md) has to be added to the whitelist of principals that are allowed to create SNSes. This is a temporary measure to make sure that no one launches an SNS before the code is ready. In the long-term, this condition will be removed. 
+3. Your cycles wallet contains enough cycles to create an SNS. Currently, this cost is 50T cycles.
+
+The SNS is new and may still have significant bugs.
+To prevent huge numbers of projects giving control of their dapps to SNSs before the SNS has been tested in production, 
+access to the SNS is limited. The decision who gets to be one of the brave first projects to launch an SNS and be added to this whitelist is decided by the NNS community by proposal.
+Later, when the SNS has a solid track record in production, this whitelist will be dropped.
+For local development, you can add your wallet to the whitelist using the following command and identity `developer-identity`:
+
+``` bash
+# This command assumes that you are using an unencrypted identity and that you are using the identity you want to deploy the SNS with.
+$(dfx cache show)/ic-admin --secret-key-pem ~/.config/dfx/identity/$(dfx identity whoami)/identity.pem --nns-url "https://localhost:$(dfx info replica-port)" propose-to-update-sns-deploy-whitelist --added-principals "$(dfx identity get-wallet)" --proposer "$DEVELOPER_NEURON_ID" --proposal-title "Let me SNS!" --summary "I am friendly."
+```
+
+Creating an SNS currently uses up to 250T cycles, and your wallet needs to supply those.
```

Comment:
```suggestion
Creating an SNS currently uses up to 180T cycles, and your wallet needs to supply those.
```

As stated above, I am not sure which number is up to date with the version we use in the demo script, but this should be the same number as above, right?